app/views/mechanic/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Mechanic</h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>Experience: <%= @mechanic.years_experience %></p>
+
+
+<p> Rides:</p>
+<% @mechanic.rides.each do|ride| %>
+  <p> <%= ride.name %></p>
+<% end %>
+
+<%= link_to 'Add Ride', "/mechanic/#{@mechanic.id}/new" , method: :get %>
+
```

Comment:
This is a much better use of HTML. Good job here. 

Again, the US did not ask for a link to a new view, making this US not fully follow the directions. Taking a look at the link path though, this is not correct. If we are retrieving a form to create a new link between a mechanic and a ride then this path should be `"/mechanics/:id/rides/new"`